Created the file 'glossary.md' for the glossary section of the handbook.

Successfulsebunya opened the pull request:
Merged the content from the Glossary into this glossary.md

Successfulsebunya committed:
Created the file 'glossary.md' for the glossary section of the handbook.

Dhruval-678 commented on glossary.md:
```diff
@@ -0,0 +1,126 @@
+## Windmill
+Adapted from [Don Quixote],(https://en.wikipedia.org/wiki/Tilting_at_windmills) a windmill in the context used on make/flow is a long term goal or vision that you never stop pursuing while attempting to gradually, steadily change a system. Actions taken toward a windmill’s goal are called tilts. Tilts over time steer change. The Flow Patrol windmills are listed [here](https://make.wordpress.org/t ...
+
+[Add new entry](https://make.wordpress.org/test/wp-admin/post-new.php?post_type=glossary)
```
@Successfulsebunya No need this kindly remove

Successfulsebunya replied:
removed

Dhruval-678 commented on glossary.md:
```diff
@@ -0,0 +1,126 @@
+## Boxed input scrolling
+Boxed input scrolling is when you can’t move the cursor through text lying beyond the confines of an input box. This sometimes happens on iOS devices, particularly in the post title and caption inputs.
+
+https://make.wordpress.org/test/tag/boxed-input-scrolling/ [#](https://make.wordpress.org/test/handbook/glossary/#boxed-input-scrolling) – [Edit](https://make.wordpress.org/test/wp-admin/post.php?post=1852&action=edit)
```
@Successfulsebunya 
Remove the edit link. Need to remove it from all

Successfulsebunya replied:
removed from all and left only the #

Successfulsebunya committed:
Update glossary.md

Resolved comments by Dhruval-678 
- removed edit links
- removed the add new entry link

Dhruval-678 commented on glossary.md:
```diff
@@ -96,31 +95,28 @@ When experiencing a theme that doesn’t provide print-styles.css, remember this
 ## Windmill
-Adapted from [Don Quixote],(https://en.wikipedia.org/wiki/Tilting_at_windmills) a windmill in the context used on make/flow is a long term goal or vision that you never stop pursuing while attempting to gradually, steadily change a system. Actions taken toward a windmill’s goal are called tilts. Tilts over time steer change. The Flow Patrol windmills are listed [here](https://make.wordpress.org/t ...
-
-[Add new entry](https://make.wordpress.org/test/wp-admin/post-new.php?post_type=glossary)
```
@Successfulsebunya 
Also remove the First published & last updated dates as well.

Successfulsebunya replied:
done

Successfulsebunya committed:
Update glossary.md

removed the first published and last updated dates

Dhruval-678 reviewed (approved):
LGTM!

Gulamdastgir-Momin reviewed (approved):
LGTM!
